src/app/recipes/recipe-edit/recipe-edit.component.html

```diff
@@ -1,131 +1,163 @@
+    <h1>
+      <label for="title"></label>
+      <input
+        id="title"
+        type="text"
+        formControlName="title"
+        placeholder="Recipe Title.."
+      />
+    </h1>
+    <button
+      class="submit-button rounded"
+      type="submit"
+      [disabled]="!recipeForm.valid"
+    >
+      Save
+    </button>
   </div>
 
-  <div>
-    <label for="desctiption"></label>
-    <input
-      id="desctiption"
-      type="text"
-      formControlName="desctiption"
-      placeholder="Recipe desctiption"
-    />
-  </div>
-  <div>
-    <label for="image">Recipe image</label>
-    <input
-      type="file"
-      id="image"
-      accept="image/png, image/jpeg"
-      (change)="updatePreviewImg($event)"
-    />
+  <div class="description-container">
+    <div class="col">
+      <label for="desctiption"></label>
+      <textarea
+        class="description-container__input"
+        id="desctiption"
+        type="text"
+        formControlName="desctiption"
+        placeholder="Recipe desctiption"
+      >
+      </textarea>
+    </div>
+
+    <div class="description-container__image-input">
+      <label for="image">Recipe image</label>
+      <input
+        type="file"
+        id="image"
+        accept="image/png, image/jpeg"
+        (change)="updatePreviewImg($event)"
+        title="uh "
+      />
+    </div>
     <img
+      class="col description-container__image"
       [src]="imagePreview"
       alt="Recipe image"
       width="250"
       *ngIf="imagePreview"
     />
   </div>
-  <!-- ingredients list -->
-  <div formArrayName="ingredients">
-    <button
-      class="btn btn-success"
-      type="button"
-      (click)="addIngredient('', 1, 'g')"
-    >
-      +
-    </button>
-    <ul>
-      <li
-        *ngFor="
-          let ingredient of getFormArray('ingredients').controls;
-          let i = index
-        "
+
+  <div class="recipe-body">
+    <!-- ingredients list -->
+    <div class="ingredients-container col-5" formArrayName="ingredients">
+      <button
+        class="ingredients-container__add-button list__add-button rounded"
+        type="button"
+        (click)="addIngredient('', 1, 'g')"
       >
-        <div [formGroup]="$any(ingredient)">
-          <label for="{{ 'ingredient_' + i }}"></label>
-          <input
-            id="{{ 'ingredient_' + i }}"
-            type="text"
-            formControlName="name"
-            placeholder="Add ingredient"
-          />
-          <label for="{{ 'ingredientQuantity_' + i }}"></label>
-          <input
-            id="{{ 'ingredientQuantity_' + i }}"
-            type="number"
-            min="1"
-            formControlName="amount"
-          />
+        Add Ingredient
+      </button>
+      <ul>
+        <li
+          *ngFor="
+            let ingredient of getFormArray('ingredients').controls;
+            let i = index
+          "
+        >
+          <div [formGroup]="$any(ingredient)" class="ingredient list__element">
```

Comment:
Is this necessary? I can't recall any scenario forcing me to do that, try to avoid this